src/main/java/uk/gov/hmcts/reform/unspec/config/ExternalTaskListenerConfiguration.java

```diff
@@ -0,0 +1,27 @@
+package uk.gov.hmcts.reform.unspec.config;
+
+import org.camunda.bpm.client.ExternalTaskClient;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+public class ExternalTaskListenerConfiguration {
+
+    private final String baseUrl;
+
+    public ExternalTaskListenerConfiguration(@Value("${feign.client.config.remoteRuntimeService.url}") String baseUrl) {
+        this.baseUrl = baseUrl;
+    }
+
+    //TODO: define a workerId, max tasks and a sensible backOffStrategy - i.e how long should I wait between long
+    // polling, currently it is the default exponential back off strategy (double the value until it reaches 60000ms).
+    // Default max tasks is 10, and lock time per task is 20000L. Do we need to think about authentication?
```

Comment:
Can we get rid of it before merging? If the questions are not answered they should be replaced by ticket reference imho.

Reply:
yeah I agree this is too much, actually chatting through some of this with Chris at the moment 👍

Reply:
pulled this out to this ticket https://tools.hmcts.net/jira/browse/CMC-737